sirepo/simulation_db.py

```diff
@@ -232,6 +232,8 @@ def _last_modified(data, path):
             data.simulationType = "warppba"
         elif data.simulationType == "fete":
             data.simulationType = "warpvnd"
+        elif data.simulationType == "ml":
```

Comment:
If we aren't carrying over old simulations then I don't think we will ever have a case where data.simulationType == "ml"